gtfs-realtime/spec/en/reference.md

```diff
@@ -170,8 +171,8 @@ Note that the update can describe a trip that has already completed.To this end,
 
 Timing information for a single predicted event (either arrival or departure). Timing consists of delay and/or estimated time, and uncertainty.
 
-*   delay should be used when the prediction is given relative to some existing schedule in GTFS.
-*   time should be given whether there is a predicted schedule or not. If both time and delay are specified, time will take precedence (although normally, time, if given for a scheduled trip, should be equal to scheduled time in GTFS + delay).
+*   delay should be used when the prediction is given relative to some existing schedule in GTFS. For added or replacement trips, the existence of the delay field indicates that there is a scheduled time, calculated by `time - delay`, at the stop for the trip, in addition to the estimated time.
```

Comment:
As I said in the other PR I'm not a fan of this. I don't think it's obvious unless you explicitly read the spec which is something I think we should avoid. Even if the spec is necessary for edge cases and details, I think the intuitive interpretation of the data should match as close as possible what is in the spec.

Reply:
Thinking out aloud, I suggested introducing `scheduledTime` in a thread (that I can't find again) in this PR.

I'm still not 100% sure it's less confusing. WDYT?

Reply:
Yes, that's what I said in the other PR too I think! That's what I would do

Reply:
I will add this field in the coming few days as it seems to be the community consensus.

Reply:
"aimed time" in the SIRI sense?

Reply:
Yes, replacing the original scheduled time in GTFS static which may have effect if the stop is a timing point.